catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/AssociationMetacardTypeImpl.java

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.data.impl;
+
+import ddf.catalog.data.AttributeDescriptor;
+import ddf.catalog.data.types.Core;
+import java.util.HashSet;
+import java.util.Set;
+
+public class AssociationMetacardTypeImpl extends MetacardTypeImpl {
```

Comment:
❗️ Can we follow the same pattern as the rest of our metacard namespaces? Usually we have a `type` API class and the impl is separated. This here is coupling the API (attribute names) and impl. We should also follow our naming conventions for attributes. https://codice.atlassian.net/wiki/spaces/CODICE/pages/77627393/Taxonomy+Guidelines

Reply:
These should also be marked experimental.